be/src/olap/rowset/rowset_meta.h

```diff
@@ -269,6 +269,21 @@ class RowsetMeta {
         return score;
     }
 
+    uint32_t get_merge_way_num() const {
+        uint32_t way_num = 0;
+        if (!is_segments_overlapping()) {
+            if (num_segments() == 0) {
+                way_num = 0;
+            } else {
+                way_num = 1;
+            }
+        } else {
+            way_num = num_segments();
+            CHECK(way_num > 0);
+        }
+        return way_num;
```

Comment:
same as score?

Reply:
The score contains an empty rowset, while the merge way does not include an empty rowset